components/esp_jpeg/Kconfig

```
@@ -0,0 +1,70 @@
+menu "JPEG Decoder"
+
+    config JD_USE_ROM
+        bool "Use TinyJPG Decoder from ROM"
+        depends on !IDF_TARGET_ESP32S2
+        default y
+        help
+            By default, Espressif SoCs use TJpg decoder implemented in ROM code. 
+            If this feature is disabled, new configuration of TJpg decoder can be used.
+            Refer to REAME.md for more details.
+
+    config JD_SZBUF
+        int "Size of stream input buffer"
+        depends on !JD_USE_ROM
+        default 512
+
+    config JD_FORMAT
+        int
+        depends on !JD_USE_ROM
+        default 0 if JD_FORMAT_RGB888
+        default 1 if JD_FORMAT_RGB565
+
+        choice
+            prompt "Output pixel format"
+            depends on !JD_USE_ROM
+            default JD_FORMAT_RGB888
+            help
+                Output format of the TJpg decoder. The component can select the output format in runtime,
+                but it depends on this choice.
+
+        config JD_FORMAT_RGB888
+            bool "RGB888 (24-bit/pix)"
+        config JD_FORMAT_RGB565
+            bool "RGB565 (16-bit/pix)"
```

Based on our discussion, should this be:
```suggestion
        config JD_FORMAT_RGB888
            bool "RGB565 and RGB888 (24-bit/pix)"
        config JD_FORMAT_RGB565
            bool "RGB565 (16-bit/pix)"
```
?

---

I don't know, what is the right. I thought, that it is only for settings TJPG decoder code. It set only output of the TJPG. Output of whole component is depend on runtime option.

---

I see. Could the labels be "Support RGB565 output" and "Support RGB565 and RGB888 output" 

and in help we could state that "Output format is selected at runtime"

?